### PR TITLE
Python 2 EOL | Updated SetDateField

### PR DESCRIPTION
**Pull request by kgal-pan**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
- Updates the `SetDateField` script to use Python 3.
- Added a Test Playbook.
- Added UTs

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
   - [x] No

### CODE REVIEW

File: Packs/CommonScripts/Scripts/SetDateField/SetDateField.yml [hunk 1]
```diff
@@ -26,6 +15,6 @@ args:
   description: "The name of the incident custom field of type date"
 scripttarget: 0
 fromversion: 5.0.0
-dockerimage: demisto/python:2.7.18.24398
+dockerimage: demisto/python3:3.10.6.33415
 tests:
-- No tests (auto formatted)
+- SetDateField-Test
```
Comment (tkatzir) [inline Packs/CommonScripts/Scripts/SetDateField/SetDateField.yml:20]: That Test Playbook does not exist

**Review by tkatzir — CHANGES_REQUESTED** [review]

File: Packs/CommonScripts/Scripts/SetDateField/SetDateField.yml (hunk 1 above)
Comment (kgal-pan) [reply Packs/CommonScripts/Scripts/SetDateField/SetDateField.yml:20]: readded.

File: Packs/CommonScripts/ReleaseNotes/1_7_28.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
 
 #### Scripts
-##### GetDockerImageLatestTag
-- Updated the Docker image to: *demisto/python3:3.10.6.33415*.
+##### SetDateField
+- Updated the docker image to *demisto/python3:3.9.5.20070*.
```
Comment (ShahafBenYakir) [inline Packs/CommonScripts/ReleaseNotes/1_7_28.md:4]: Why are we editing old rn?

**Review by ShahafBenYakir — COMMENTED** [review]
nice

File: Packs/CommonScripts/ReleaseNotes/1_7_28.md (hunk 2 above)
Comment (kgal-pan) [reply Packs/CommonScripts/ReleaseNotes/1_7_28.md:4]: At the time I was working on this PR the version was 1.7.28. I'll regenerate them to fix this.

**Pull request closed without merging**